proto/p4/p4runtime.proto

```diff
@@ -384,10 +398,10 @@ message MulticastGroupEntry {
 // packet_length_bytes field is 0, no truncation on the clone(s) will be
 // performed.
 message CloneSessionEntry {
-  uint64 session_id = 1;
+  uint32 session_id = 1;
   repeated Replica replicas = 2;
-  uint64 class_of_service = 3;
-  uint64 packet_length_bytes = 4;
+  uint32 class_of_service = 3;
+  uint32 packet_length_bytes = 4;
```

Comment:
Isn't the best practice to use int32 here? I'm not totally sure but I think I remember @wmohsin mentioning something like this.